File: js/singles/struct.js
function Address() {    // actually, this is a function that
    this.name = "";     // returns an object (filled struct) 
    this.street = "";   // with the default values that also
    this.postcode = 0;  // define the type
    this.city = "";
}

var address = new Address(); // the keyword 'new' does the trick 
address.name = "S. Error";   // as it creates the struct address
address.street = "Bus 0";
address.postcode = 6502;
address.city = "Processor";
console.log(address.street);

File: c/singles/struct.c
#include <stdio.h>
#include <string.h>

struct Address {
    char name[50];          /* attention only space for 50 characters */
    char street[50];
    unsigned int postcode;
    char city[50];
};

int main() {
    struct Address address;

    strcpy(address.name, "S. Error");   /* error prone, second string */
    strcpy(address.street, "Bus 0");    /* must have less 50 characters */
    strcpy(address.city, "Processor");  /* function copies the characters*/
    address.postcode = 6502;

    printf("%s", address.name);
    return 0;
}